docs/WritingTests.asciidoc

```diff
@@ -428,24 +428,22 @@ Children B, C, D can run and finish anytime, but A must run until all B, C, D fi
 
 === Test synchronization and locking API
 
-OpenQA provides locking server through lock API. To use lock API import +lockapi+ package (_use lockapi;_) in your test file.
-Lock API provides functions: +mutex_create+, +mutex_lock+, +mutex_unlock+, +mutex_wait+. Each of these functions take at least one parameter: name of the lock. Note that lock name can't contain "-" character.
-Locks are associated with caller's job - locks can't be unlocked by different job then the one who locked the lock.
+OpenQA provides a locking server. To use it in your test files import +lockapi+ package (_use lockapi;_). It provides the following functions: +mutex_create+, +mutex_lock+, +mutex_unlock+, +mutex_wait+
```

Comment:
I find term "locking server" confusing. When I hear this I am thinking about some entity which I need to start or even install somehow. 
```suggestion
OpenQA provides a locking API. To use it in your test files import +lockapi+ package (_use lockapi;_). It provides the following functions: +mutex_create+, +mutex_lock+, +mutex_unlock+, +mutex_wait+
```

Reply:
This is just the term as it was in the documentation before. I've just added an article and stopped the sentence here. In fact this *is* a server (part of the openQA web UI server). But I can change it to "locking mechanism".

Reply:
In my understanding this documentation suppose to explain to **user** of lock API how actually to use it. **User** don't care about how it is implemented under the hood. He/she just needs to know how to use it.